Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -109,6 +110,9 @@ inline OffsetDistance forwardInheritedValue(const OffsetDistance& value) { auto
 inline OffsetPath forwardInheritedValue(const OffsetPath& value) { auto copy = value; return copy; }
 inline OffsetPosition forwardInheritedValue(const OffsetPosition& value) { auto copy = value; return copy; }
 inline OffsetRotate forwardInheritedValue(const OffsetRotate& value) { auto copy = value; return copy; }
+inline Position forwardInheritedValue(const Position& value) { auto copy = value; return copy; }
```

Comment:
Reading all these functions, I was wondering how they are different from just:
```suggestion
inline Position forwardInheritedValue(const Position& value) { return value; }
```
I suppose copying into a local variable first must have some effect, but what is it?

Reply:
Probably none, and I’ve just been copying the idiom.

Reply:
Ok, so this is fun. You (!) add this idiom in https://github.com/WebKit/WebKit/commit/4a44702db4c4cfbbcf8e51b1e505ef28825bfb4c#diff-aca64cace236dc9180bbb97adba67b3b91660953ab9b0920d9bdff5635897870R61, but I can't escape blameless, as I reviewed it!

Reply:
Only 8 years ago

Reply:
How could we forget!